packages/wp-now/README.md

```diff
@@ -89,211 +91,234 @@ cd wordpress-plugin-or-theme
+
+-   `wp-now` handles the entire WordPress installation for you. Just run the `wp-now start` command.
+-   `wp-now` works across all desktop platforms (Mac, Linux, Windows).
+-   `wp-now` does not set up custom domains for you.
+-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed (see item #1 above).
+-   `wp-now` allows you to switch the WordPress version with `wp-now start --wp=VERSION_NUMBER`(replace `VERSION_NUMBER` with the actual WordPress version).
+
+### wp-env
 
-Some similarities between `wp-env` and `wp-now` to be aware of:
+Here's what you need to know if you're migrating from `wp-env`:
 
--   no support for custom domains or SSL;
--   `plugin`, `themes` and index modes are available on `wp-env` and `wp-now`;
--   deployments are not possible with neither `wp-env`, nor `wp-now`;
--   possible to switch easily the PHP version
+-   `wp-now` supports non-WordPress projects.
+-   `wp-now` does not require Docker.
+-   `wp-now` does not include Jest for automatic browser testing.
 
 ## Contributing
 
 We welcome contributions from the community!
 
-In order to contribute to `wp-now`, you'll need to first install a few global dependencies:
+### Code contribution
 
--   Make sure you have `nvm` installed. If you need to install it first,
-    [follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
--   Install `nx` by running `npm install -g nx`.
-
-Once the global dependencies are installed, you can start using the repo:
+Clone the repository, install the dependencies, build the project, and start `wp-now` in `preview` mode:
 
 ```bash
 git clone git@github.com:WordPress/playground-tools.git
 cd playground-tools
-nvm use
 npm install
 npm run build
-nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
+npx nx preview wp-now start
 ```
 
-If you'd like to make the `wp-now` executable globally available when using this installation method, run `npm link`. It's not particularly reliable, however.
-
-## Testing
-
 To run the unit tests, use the following command:
 
 ```bash
-nx test wp-now
-```
-
-## Publishing
-
-The `wp-now` package is part of a larger monorepo, sharing its space with other sibling packages. To publish the `wp-now` package to npm, you must first understand the automated release process facilitated by lerna. This process includes automatically incrementing the version number, creating a new tag, and publishing all modified packages to npm simultaneously. Notably, all published packages share the same version number.
-
-Each package identifies a distinct organization in its `package.json` file. To publish the `wp-now` package, you need access to the `@wp-now` npm organization.
-
-To initiate the publishing process for `wp-now`, execute the following commands:
-
-```bash
-npm login # this is required only once and it will store the credentials in ~/.npmrc file.
-npm run build
-npm run release:wp-now
+npx nx test wp-now
```

Comment:
last thought is there are a lot of changes here, TYVM for all that @ironnysh!!! 🙌

If it's alright with you all, I'm gonna pull down the changes when we review more, and test everything.

See if any other pitfalls come up, discuss more. There's so much here my first thought is that more pages may help to simplify the info or break it out in to micro-sections

Reply:
> more pages may help to simplify the info 

This should probably be in the Playground documentation site (there's a [Local development chapter](https://wordpress.github.io/wordpress-playground/local-development/wp-now) now), but the README should remain a single page.